CMakeLists.txt

```diff
@@ -80,6 +81,18 @@ if (CMAKE_SYSTEM MATCHES "Darwin")
     set(EXTRA_LIBS ${CORESERVICES_LIB})
 endif()
 
+find_program(EDITBIN editbin)
```

Comment:
shouldn't this be limited to Windows only. Suppose an `editbin` tool exists on Linux, it probably does something different...

Reply:
It is a possibility, I thought about it, I didn't think it was necessary, but it might happen that other OS have also an editbin.